### DISCUSSION TITLE
Can src reference byte array?

### DISCUSSION THREAD

**Discussion by gilm0079** [CropperBlazor/Cropper.Blazor#326] [Q&A]
The examples look like the src is a string to a image URL. For blazor wasm, can cropper reference an in memory location like a byte array? I was trying to see if this component would work in wasm to allow profile pic cropping after uploading an image. The image in this case is uploaded into wasm memory (not yet moved to the server).

**Comment by MaxymGorn** [comment]
Hi @gilm0079. Bases on this logic: https://github.com/CropperBlazor/Cropper.Blazor/blob/6cf1b2678b261867f78b1e982faa2ea9cd7d8634/src/Cropper.Blazor/Cropper.Blazor/wwwroot/cropperJsInterop.js#L166  & example: https://cropperblazor.github.io/examples/replacing#simple-usage, you should use your blob from bytes and use created url:
```
const blob = new Blob([arrayBuffer]);
URL.createObjectURL(blob)
```
Don't forget to revoke object url when it no needed
